tests/ReadAttributes/basic_test.ll

```diff
@@ -0,0 +1,60 @@
+; RUN: test `opt -load LLVMReadAttributes.so -ReadAttributes %s -o %t1.ll 2>&1 | grep -c 'tojit_f'` = 1
+
+; ModuleID = 'basic_test.c'
```

Comment:
On Tue, Feb 09, 2016 at 05:05:33AM -0800, Merlini Adrien wrote:
> I had no better idea (and too lazy to log:P), do you have any solutions?

you could add stat support (see the tutorial we made !) and pass the
flag to opt, so that you could check the stat is ok.